Command: /Users/jpedroriverop/.espressif/python_env/idf5.5_py3.9_env/bin/python /Users/jpedroriverop/esp/esp-idf/tools/idf_monitor.py -p /dev/cu.usbserial-0001 -b 115200 --toolchain-prefix xtensa-esp32-elf- --target esp32 --revision 0 /Users/jpedroriverop/proyectos/proyecto-ruido-iot/hardware/simulador_ruido/build/simulador_ruido.elf /Users/jpedroriverop/proyectos/proyecto-ruido-iot/hardware/simulador_ruido/build/bootloader/bootloader.elf -m '/Users/jpedroriverop/.espressif/python_env/idf5.5_py3.9_env/bin/python' '/Users/jpedroriverop/esp/esp-idf/tools/idf.py' '-p' '/dev/cu.usbserial-0001'
[0;33m--- esp-idf-monitor 1.6.2 on /dev/cu.usbserial-0001 115200[0m
[0;33m--- Quit: Ctrl+] | Menu: Ctrl+T | Help: Ctrl+T followed by Ctrl+H[0m
[0;33m--- Connection to /dev/cu.usbserial-0001 failed. Available ports:
--- /dev/cu.debug-console
--- /dev/cu.usbmodemSN234567892[0m
[0m
